ethcore/src/client/client.rs

```diff
@@ -439,7 +439,10 @@ impl<V> BlockChainClient for Client<V> where V: Verifier {
 		};
 		// that's just a copy of the state.
 		let mut state = self.state();
-		let sender = try!(t.sender());
+		let sender = try!(t.sender().map_err(|e| {
+			warn!(target: "evm", "Transaction mallformed: {:?}", e);
+			ExecutionError::TransactionMallformed
```

Comment:
You don't want to wrap the original error inside `TransactionMalformed`? Maybe at least a string representation of the error to have some more details?